manifests/worker.pp

```diff
@@ -206,14 +207,14 @@
   }
 
   cron { "${name}-logrotate":
-    command => "/usr/bin/find /home/${name}/dlrn-logs/*.log -mtime +2 -exec rm {} \;",
+    command => "/usr/bin/find /home/${name}/dlrn-logs/*.log -mtime +2 -exec rm {} \\;",
```

Comment:
Yes, it's tested. Without the double \ , rake validate fails although the cron job is ok. With double backslash, the actual cron job is correctly created:

0 4 * * * /usr/bin/find /home/centos-master/dlrn-logs/*.log -mtime +2 -exec rm {} \;

(taken from an actual puppet apply.